scripts/poe/lib/validate.js

```diff
@@ -1,21 +1,31 @@
-const {log, error, mkdir, extractPlaceholders} = require('./utils');
+const utils = require('./utils');
 const fs = require('fs');
-const path = require('path');
+const {
+  checkTranslations,
+  TranslationException
+} = require('translation-checker');
+const MessageFormat = require('messageformat');
+
+const MFORMAT = new MessageFormat('en');
+const transErrorsMsg = MFORMAT
+  .compile('There {ERRORS, plural, one{was 1 error} other{were # errors}} trying to compile');
+const transEmptyMsg = MFORMAT
+  .compile('There {EMPTIES, plural, one{was 1 empty translation} other{were # empty translations}} trying to compile');
```

Comment:
Now that this is the only use of `MessageFormat` it'd be better to drop it altogether and just using hardcoded strings for error reporting.